src/utilities.ts

```diff
@@ -30,15 +29,6 @@ export let globalTestInterfacePath: string | undefined = undefined;
 let globalCrc32Path: string | undefined = undefined;
 let globalPathToSupportFiles: string | undefined = undefined;
 
-export function setPaths(
```

Comment:
We use `setPaths` for unit tests, but from `pythonUtilities`. I guess it's ok to remove it from here

Reply:
Ohh, wondered why this was not called :)  If you need this, we syould add it back into clicast.ts, want me to do that?

Reply:
I checked now. Tests use `setPaths` from `server/pythonUtilities.ts`. No need to add this back to `clicast.ts`